### ISSUE TITLE
Method for injecting trace parent into curl

### ISSUE THREAD

**Issue by jeeftor** [equinix-labs/otel-cli#306]
I wrote my own cli client for Otel and then found this one ... is there a way to inject the trace parent from an `exec` into the curl headers so I can see the full round trip of my existing tracing with this command?

**Comment by tobert** [comment]
There's not an easy way to do it right now. I had an old issue for this that I'll work on now that I have a request for it.

The way I've done it so far is with a TRACEPARENT environment variable.
```shell
traceparent_file=$(mktemp)
otel-cli span --name "first span in the trace" --tp-carrier $traceparent_file --tp-export --endpoint https://collector
. $traceparent_file
otel-cli exec --endpoint https://collector -- curl -X "traceparent: $TRACEPARENT" https://site.com/file
```

**Comment by jeeftor** [comment]
Ended up just writing my own client

**Comment by tobert** [comment]
Released 0.4.3 with the injection feature. Sorry the timing didn't work out @jeeftor, and thanks for the bump to get this feature in.
